l3experimental/xcoffins/xcoffins.dtx

```diff
@@ -377,7 +377,7 @@
 %   processes are contrasted. In both cases, the small red coffin has been
 %   aligned with the large grey coffin. In the left-hand illustration,
 %   the \cs{JoinCoffins} function was used, resulting in an expanded
-%   bounding box. In contrast, on the right \cs{AttachCoffin} was used,
+%   bounding box. In contrast, on the right \cs{JoinCoffins*} was used,
```

Comment:
Extract star out of csname?

```suggestion
%   bounding box. In contrast, on the right \cs{JoinCoffins}|*| was used,
```

Reply:
I don't know what is the recommended way to spell it.

Reply:
`\cs{foo*}`  puts `\foo*` but the star is really syntax (like an optional command) so we normally do not put this into the index.

Reply:
OK. Should I create a new PR?

Reply:
no. rebase and update the existing one